packages/packages/top-bar/src/components/__tests__/top-bar.test.tsx

```diff
@@ -1,10 +1,12 @@
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import TopBar from '../top-bar';
 
 describe( '@elementor/top-bar TopBar component', () => {
 	it( 'should render elementor logo', () => {
-		const { queryByAltText } = render( <TopBar /> );
+		const { queryAllByRole } = render( <TopBar /> );
 
-		expect( queryByAltText( 'Elementor Logo' ) ).toBeTruthy();
```

Comment:
we still need the logo assertion somehow

Reply:
I've changed the test a bit, and currently, there is no real functionality anyway.

Reply:
We still need to test that the logo appears

Reply:
Done.